### PR TITLE
Transcription fixes

### LINKED ISSUE #50: Transcripts don't support markdown
Though the [docs](https://github.com/ryanvilbrandt/comic_git/wiki/Extra-Features#transcripts) say transcripts support markdown, this wasn't the case when I tested. I added support by modifying the `get_transcripts` function in `build_site.py`: (lines 231-233)

```
  with open(path, "rb") as f:
      transcripts[language] = f.read().decode("utf-8").replace("\n", "<br>\n")
      transcripts[language] = MARKDOWN.convert(transcripts[language])
```

Happy to open a PR for this change but wanted to confirm first this is a bug.

### PR DESCRIPTION
**Pull request by shirgoldbird**
Fixes #50, #51, and #52.

I've tested this locally and believe it won't cause any regressions. Please give it a test yourself and let me know what you think!

### CODE REVIEW

File: src/scripts/build_site.py [hunk 1]
```diff
@@ -83,7 +83,7 @@ def read_info(filepath, to_dict=False):
 
 def get_option(comic_info: RawConfigParser, section: str, option: str, option_type: type=str, default: str=None) -> str:
     if comic_info.has_section(section):
-        if comic_info.has_option(section, option):
+        if comic_info.has_option(section, option) and comic_info.get(section, option):
```
Comment (ryanvilbrandt) [inline src/scripts/build_site.py:86]: What is a use-case where you found this fix was needed? This change would make it impossible for a user to supply an empty string for a config option, if that config option had a hardcoded default. That said, I can't think of a situation where a user would want an empty string as their config value, but still...

File: src/js/transcript.js [hunk 2]
```diff
@@ -3,6 +3,13 @@ export function init() {
     let languageList = document.getElementById("language-select");
     let transcriptList = document.getElementById("active-transcript");
     let transcripts = transcriptList.children;
+
+    // don't show language picker if we only have one language
+    if (!languageList) {
+        transcripts[0].style.display = "block";
+        return;
+    }
```
Comment (ryanvilbrandt) [inline src/js/transcript.js:11]: I'll look into if there's a way to move this logic into the template file.

File: src/scripts/build_site.py [hunk 3]
```diff
@@ -228,7 +228,8 @@ def get_transcripts(comic_folder: str, comic_info: RawConfigParser, page_name: s
             continue
         language = os.path.splitext(os.path.basename(path))[0]
         with open(path, "rb") as f:
-            transcripts[language] = f.read().decode("utf-8").replace("\n", "<br>\n")
+            transcripts[language] = f.read().decode("utf-8")
+            transcripts[language] = MARKDOWN.convert(transcripts[language])
```
Comment (ryanvilbrandt) [inline src/scripts/build_site.py:232]:
```suggestion
            transcripts[language] = MARKDOWN.convert(f.read().decode("utf-8"))
```

**Review by ryanvilbrandt — CHANGES_REQUESTED** [review]

**Comment by ryanvilbrandt** [comment]
I've redone a lot of this work in a new pull request in #58, partially to make slight changes to your work, and partially so I could easily test the changes myself.

Please take a look at that PR and let me know if I missed anything.

File: src/scripts/build_site.py (hunk 1 above)
Comment (shirgoldbird) [reply src/scripts/build_site.py:86]: Sounds good! Removing it from the default file is a fine fix too if you think this may cause unexpected behavior.

**Comment by ryanvilbrandt** [comment]
Handled by #58

**Pull request closed without merging**